nestor_api/lib/k8s/builders.py

```diff
@@ -36,6 +34,30 @@ def build_yaml(deployment_config: dict, templates_path: str, tag_to_deploy: str)
     return "\n".join(yaml_sections)
 
 
+def build_ingress_yaml(deployment_config: dict, templates: dict) -> str:
+    """Builds the k8s yaml corresponding to the provided configuration."""
+    app_name, sanitized_process_name, metadata_name = get_sanitized_names(deployment_config, "web")
```

Comment:
```suggestion
    app_name, sanitized_process_name, metadata_name = get_sanitized_names(deployment_config, WEB_PROCESS_NAME)
```

Maybe use the defined `WEB_PROCESS_NAME = "web"`

Reply:
oops 😆

Reply:
I will pass it as an argument in `build_ingress_yaml` to prevent introducing circular dependencies between the two modules.